### ISSUE TITLE
test configurations, forcing

### ISSUE THREAD

**Issue by eclare108213** [CICE-Consortium/Icepack#79]
We need a description of each of the forcing data sets, and the 'test configurations' section needs to be updated or removed from the User Guide (section 1, Implementation).  This section generally needs more information and/or updating.

**Comment by duvivier** [comment]
Elizabeth, I've moved this to the testing section of the User's guide and updated it to reflect the info we have on the Wiki. When I do a PR I'll have you review it and we can see if we want to add more info.

**Comment by duvivier** [comment]
Fixed with PR #104

**Comment by eclare108213** [comment]
We need more detailed information in the docs.  I'm thinking that for each of our forcing options in Icepack (default, clim, cfs, ispol, nice) we need to include the data source and characteristics such as how frequent it is, which variables are included, known biases (if we know them), assumptions, and references to papers associated with the data, where users could go for more info.  Some of this info is cataloged here:  https://docs.google.com/document/d/1FQISNl-d3YUjZO7XmKVKANguH2FnT9wuB3IEQqdB7ls/edit#bookmark=id.kul63twvehqw
I'll ask our data contributors to please help get this into the formal docs:  @MichaelWinton @rallard77 @njeffery
@dabail10 you also added info for the defaults (hardwired summer, winter, etc) - I'm not sure where those originated or if we want to discuss them, but we should at least note that they are in there for various seasons.  Thx

**Comment by MichaelWinton** [comment]
I tried to address this for the climatological forcing in the driver design document ( https://docs.google.com/document/d/1FQISNl-d3YUjZO7XmKVKANguH2FnT9wuB3IEQqdB7ls/edit#heading=h.rqw060ar17rm ).  Was this not what was wanted?

**Comment by eclare108213** [comment]
@MichaelWinton I'm sorry for not being clear.  What you've added to the google doc is perfect:  I'm looking for is a short paragraph, maybe just a few sentences, with content and style as you would use in a journal publication to describe data being used.  The paragraph needs to be put into the primary documentation, https://cice-consortium.github.io/Icepack.
@duvivier I'm having trouble figuring out which rst file should be edited -- it looks like the forcing appears in multiple places, depending on whether I'm looking at the rst files directly (in the code repo) or the github.io pages.  If everyone writes their paragraphs into the google doc, could you copy them into the main doc and update the html?  
Thanks everyone.

**Comment by duvivier** [comment]
Yes, I can add things into main doc and update html. I got Rick's info already and will work on adding Mike's.

**Comment by duvivier** [comment]
Question: I think we'd decided to move the forcing info to section 3.3 (Testing) instead of 3.2 (running) in the user's guide. However, we hadn't updated the html for that change yet. Given that we're going to have the forcing info for all the options, do you think it makes sense to keep it at 3.2 instead of 3.3?
https://cice-consortium.github.io/Icepack/user_guide/ug_running.html#forcing-data

**Comment by eclare108213** [comment]
I think it makes more sense in section 3.2, running Icepack, rather than testing.  The testing section seems to be more about the scripts specific to setting up tests, while 'running' is more about getting the broader run environment configured.

**Comment by duvivier** [comment]
Agreed, I will move it there.

Also, the Google Doc has SHEBA as an option. But as of right now we don't have that as a namelist option at the moment. Should it be included? Does someone still need to fill this in?

https://docs.google.com/document/d/1FQISNl-d3YUjZO7XmKVKANguH2FnT9wuB3IEQqdB7ls/edit#

**Comment by eclare108213** [comment]
Ask @dabail10 ;)

**Comment by duvivier** [comment]
Ok. :)  I know he was working on the ridging dataset from SHEBA.

**Comment by dabail10** [comment]
Right. There will be an ocn_data_type = "SHEBA" for this. Long term we might have an atm_data_type as well.

**Comment by eclare108213** [comment]
This seems to be mostly done except including a note in the forcing documentation that we also have different seasons (set as a namelist option) for the default values set in the code.
Is anything else still missing on this topic?

**Comment by duvivier** [comment]
The SHEBA info isn't done.

**Comment by dabail10** [comment]
I updated the SHEBA forcing information in the documentation and added something about default_season. I just issued PR #146 with this in it.

**Comment by rallard77** [comment]
We should add a description in the atmospheric forcing documentation about the "default" case which can be used with the "spring", "summer", and "winter" options.
